zebrad/src/components/mempool/tests.rs

```diff
@@ -1,8 +1,10 @@
 use std::pin::Pin;
 
-use tower::ServiceExt;
+use tower::{BoxError, ServiceExt};
```

Comment:
Suggestion (4/4): Can we use `zebrad::BoxError` instead?

Reply:
Done.